apps/shipments/serializers.py

```diff
@@ -115,6 +129,10 @@ class JSONAPIMeta:
         included_resources = ['ship_from_location', 'ship_to_location', 'bill_to_location',
                               'final_destination_location', 'load_data', 'device']
 
+    def validate_geofences(self, geofences):
+        # Deduplicate list
+        return list(set(geofences))
```

Comment:
I think we could use `shipchain_common.test_utils.validate_uuid4` here to validate the UUID type.